src/modules.js

```diff
@@ -76,7 +76,9 @@ var LibraryManager = {
       libraries.push('library_exceptions_stub.js');
     }
 
-    if (!MINIMAL_RUNTIME) {
+    if (MINIMAL_RUNTIME) {
+      libraries.push('library_strings.js');
```

Comment:
it seems odd to have this included only in minimal runtime. perhaps a comment can clarify why this one is special?

Reply:
check